auth0/2-add-github-teams.js

```diff
@@ -7,7 +7,8 @@ function (user, context, callback) {
   })[0];
   var access_token = github.access_token;
   request.get({
-      url: "https://api.github.com/user/teams",
+      baseUrl: "https://api.github.com/",
+      uri: "/user/teams?per_page=100",
```

Comment:
This will fix the issue for people on 30-100 teams, but we might still have an issue if people are on more teams than that?

Reply:
that's correct. this is a quick fix. if someone wants to spend more time figuring out javascript i think all the current rules could be cleaned up and less error prone